core-meetings/2020-04-15-issue-management-meeting.md

```diff
@@ -0,0 +1,46 @@
+# 2020-04-15 React Native Issue Management Meeting
+
+## Agenda:
+- Discuss common patterns observed in offline triage
+- Go through Needs: Issue Manager Attention issues
+- Go through Needs: Triage issues
+- Go through Needs: Attention issues
+- Discuss areas of improvement
+- Create action items
+
+## Notes
+
+- No issues flagged [Needs: Issue Manager Attention](https://github.com/facebook/react-native/labels/Needs%3A%20Issue%20Manager%20Attention) this week
+- Lots of issue traffic from upgrade issues to 0.62
+  - [Type: Upgrade Issue](https://github.com/facebook/react-native/labels/Type%3A%20Upgrade%20Issue) will redirect to the upgrade helper and close it.
```

Comment:
Should redirect to Upgrade Support (https://github.com/react-native-community/upgrade-support).

```suggestion
  - [Type: Upgrade Issue](https://github.com/facebook/react-native/labels/Type%3A%20Upgrade%20Issue) will redirect to the Upgrade Support repository and close it.
```